File: test/fsharpTests/CharTest/fromStringTest.fs
testCase "fromString("a")" 
<| fun _ -> 
    let expected = Some('a')
    Expect.equal expected (Char.fromString "a") "error"
testCase "fromString("abc")" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Char.fromString "abc") "error"
testCase "fromString("")" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Char.fromString "") "error"

File: test/ocamlTests/CharTest/fromStringTest.ml
test "fromString("a")" (fun () -> expect (Char.fromString "a") |> toEqual (let open Eq in option char) Some('a')) ; 
test "fromString("abc")" (fun () -> expect (Char.fromString "abc") |> toEqual (let open Eq in option char) None) ; 
test "fromString("")" (fun () -> expect (Char.fromString "") |> toEqual (let open Eq in option char) None) ; 
